smarts/core/renderer.py

```diff
@@ -76,6 +111,14 @@ def __new__(cls):
             it.init()
         return it
 
+    @classmethod
+    def set_rendering_mode(cls, debug_mode: DEBUG_MODE):
+        cls._debug_mode = debug_mode
```

Comment:
This will have no effect after the `_ShowBaseInstance` object has been created, right?
If not, maybe add an assert here if called too late.

Reply:
I have set it to affect at any point.